File: app/src/main/java/no/nav/tilbakemeldingsmottak/consumer/joark/domain/OpprettJournalpostResponseTo.java
package no.nav.tilbakemeldingsmottak.consumer.joark.domain;

import lombok.Builder;
import lombok.Value;

@Value
@Builder
public class OpprettJournalpostResponseTo {
    private String journalpostId;
    private String journalstatus;
    private String melding;
}

File: app/src/main/kotlin/no/nav/tilbakemeldingsmottak/consumer/joark/domain/OpprettJournalpostResponseTo.kt
package no.nav.tilbakemeldingsmottak.consumer.joark.domain

// Swagger: https://dokarkiv.dev.intern.nav.no/swagger-ui/index.html
// Java: https://github.com/navikt/dokarkiv/blob/master/journalpost/src/main/java/no/nav/dokarkiv/journalpost/v1/api/opprettjournalpost/OpprettJournalpostResponse.java
data class OpprettJournalpostResponseTo(
    val journalpostId: String,
    val journalstatus: String? = null, // FIXME: @Deprecated, ta i bruk journalpostferdigstilt i stedet
    val melding: String? = null // @Hidden i swagger
)
